app/views/visual_groups/_form.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= form_with(model: [@visual_model, @visual_group], local: true,
+              id: "visual_group_form") do |form| %>
+  <% if visual_group.errors.any? %>
+    <div id="error_explanation">
+      <h2>
+	<%= pluralize(visual_group.errors.count, "error") %>
+	prohibited this visual_group from being saved:
+      </h2>
+
+      <ul>
+        <% visual_group.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <div class="field">
+      <%= form.label :name %>
+      <%= form.text_field(:name, size: 40) %>
+      <%= :VISUAL_GROUP.t %>
+    </div>
+
+    <div class="field">
+      <%= form.label :description %><br>
+      <%= form.text_area(:description, cols: 60, rows: 10) %>
+    </div>
+
+    <div class="field">
```

Comment:
Suggest adding Bootstrap class "form-group" on these label/field combos (where there is currently "field").

https://getbootstrap.com/docs/3.4/css/#forms